### DISCUSSION TITLE
keyboard and track pad settings

### DISCUSSION THREAD

**Discussion by kit2katbar** [Valkirie/HandheldCompanion#793] [Q&A]
Hi I was wondering I'd there was a way to use the default SteamOS/Steam Controller keyboard and trackpads with HHC?

I been finding myself using my steamdeck on the go wanted an easier time having to type things. It's very cumbersome to hold it like a handheld then switch to holding it like a fat tablet every time I need to type something. Also, I've always love the circle motion you had to do to scroll up and down with the default steam controls. 

Any way this is possible with HHC? I've had to uninstall HHC just to get those functionality back.

**Comment by CasperH2O** [comment]
@kit2katbar , by HHC do you mean Handheld Companion or some other application?

Can you share some more information about what you mean with SteamOS/Steam controller keyboard and trackpads? HC has support for the Steam Controller (Gordon) as an external controller. What do you mean with circle motion?
